### PR TITLE
Updated Request investigation content

### PR DESCRIPTION
**Pull request by kelvinkipruto**
This PR Updates the `request-investigation` page with new content

### CODE REVIEW

**Comment by kilemensi** [comment]
Please add screenshots, etc. @kelvinkipruto

File: src/app/request-investigation/request-investigation.component.html [hunk 1]
```diff
@@ -1,24 +1,20 @@
 <div class="header">
-  <h1 class="title">Investigative Support</h1>
+  <h1 class="title">Unmask the trolls</h1>
   <p class="subtitle">
-    Let our experts uncover the trolls behind your online harassment.
+    How our investigators can help
```
Comment (kilemensi) [inline src/app/request-investigation/request-investigation.component.html:4]: **How our investigators can help** or _How can our investigators help?_

File: src/app/request-investigation/request-investigation.component.html [hunk 2]
```diff
@@ -1,24 +1,20 @@
...
   </p>
   <img src="/request_investigation_illustration.svg" />
 </div>
 
 <div class="content-wrapper request-investigation-page">
   <div class="content">
-    <div class="request-content">
+    <!-- <div class="request-content">
```
Comment (kilemensi) [inline src/app/request-investigation/request-investigation.component.html:11]: Don't comment out code, delete if no longer needed.

File: src/app/request-investigation/request-investigation.component.scss [hunk 3]
```diff
@@ -124,13 +128,13 @@
       }
 
       .name {
-        line-height: math.div(27, 22);
+        // line-height: math.div(27, 22);
```
Comment (kilemensi) [inline src/app/request-investigation/request-investigation.component.scss:131]: Delete if not needed.

**Review by kilemensi — COMMENTED** [review]
👍🏽 

----


There seems to be a lot of space between paragraphs. Is this intentional?

Screenshot 2022-11-28 at 16-56-53 Request and Investigation - FeedShield

File: src/app/request-investigation/request-investigation.component.html (hunk 1 above)
Comment (kelvinkipruto) [reply src/app/request-investigation/request-investigation.component.html:4]: Perhaps it is a typo, but that is what is in the requested content change

Comment (kilemensi) [reply src/app/request-investigation/request-investigation.component.html:4]: Cool, please confirm.

Comment (kelvinkipruto) [reply src/app/request-investigation/request-investigation.component.html:4]: @kilemensi Confirmed this. It is correct as is.

**Review by kilemensi — APPROVED** [review]
🚀

**Pull request merged**